src/Symfony/Bridge/Monolog/Logger.php

```diff
@@ -23,41 +23,41 @@
 class Logger extends BaseLogger implements LoggerInterface, DebugLoggerInterface
 {
     /**
-     * @deprecated since 2.2, to be removed in 3.0. Use emergency() which is PSR-3 compatible.
+     * @deprecated since version 2.2, to be removed in 3.0. Use emergency() which is PSR-3 compatible.
      */
     public function emerg($message, array $context = array())
     {
-        trigger_error('The emerg() method of the Monolog Logger was removed. You should use the new method emergency() instead, which is PSR-3 compatible.', E_USER_DEPRECATED);
+        trigger_error('The '.__METHOD__.' method of the Monolog Logger was removed. Use the emergency() method instead, which is PSR-3 compatible.', E_USER_DEPRECATED);
```

Comment:
This message should be adjusted saying that the method is deprecated (it comes from the deprecated Symfony LoggerInterface actually)